### PR TITLE
Added vera scene documentation

### PR DESCRIPTION
**Pull request by alanfischer**
**Description:**
This adds documentation for vera scenes

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#10424

## Checklist:

  - [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`. 
  - [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/scene.vera.markdown [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: "Vera Scene"
+description: "Instructions how to integrate Vera Scenes into Home Assistant."
+date: 2015-10-20 21:00
```
Comment (frenck) [inline source/_components/scene.vera.markdown]: Incorrect date?

File: source/_components/scene.vera.markdown [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: vera.png
+ha_category: Scene
+ha_iot_class: "Local Push"
+ha_release: pre 0.7
```
Comment (frenck) [inline source/_components/scene.vera.markdown]: The version seems unlikely... 😉

File: source/_components/scene.vera.markdown [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: "Vera Scene"
+description: "Instructions how to integrate Vera Scenes into Home Assistant."
```
Comment (frenck) [inline source/_components/scene.vera.markdown]: Typo: `Instructions on how to`

**Comment by frenck** [comment]
Thank you for updating the PR @alanfischer 🏅 
Can be merged as soon as the parent PR gets merged.

File: source/_components/scene.vera.markdown [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+description: "Instructions on how to integrate Vera Scenes into Home Assistant."
+date: 2017-11-20 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: vera.png
+ha_category: Scene
+ha_iot_class: "Local Push"
+ha_release: 0.59.0
```
Comment (MartinHjelmare) [inline source/_components/scene.vera.markdown]: "0.60"

**Comment by alanfischer** [comment]
The ha_release version is now updated.

File: source/_components/scene.vera.markdown [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+description: "Instructions on how to integrate Vera Scenes into Home Assistant."
+date: 2017-11-20 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: vera.png
+ha_category: Scene
+ha_iot_class: "Local Push"
+ha_release: 0.60.0
```
Comment (MartinHjelmare) [inline source/_components/scene.vera.markdown]: It should be `"0.60"`, with quotes.

Comment (alanfischer) [reply source/_components/scene.vera.markdown]: Ok, thanks!

**Pull request merged**